language-service/src/yamlLanguageService.ts

```diff
@@ -93,6 +94,12 @@ export interface SchemaConfiguration {
 	schema?: JSONSchema;
 }
 
+export interface YamlNodeInfo {
```

Comment:
Can you move this to yamlTraversal ? I think we export every file in this repo anyway so you can add yamlTraversal to language-service/src/index.ts  and it will be exported too.